docs/get-started/getting-started-nextjs.mdx

```diff
@@ -71,69 +71,78 @@ git --version
 
 3. Open the terminal (**View > Terminal**) and clone the example app, change to the app's directory, and install the app's dependencies.
 
-  ```bash
-  git clone https://github.com/nextauthjs/next-auth-example.git
-  cd next-auth-example
-  npm install
-  ```
+```bash
+git clone https://github.com/nextauthjs/next-auth-example.git
+cd next-auth-example
+npm install
+```
```

Comment:
nit: Indents after lists are usually intentional, as they affect how content is displayed. I presume a prettifier did this automatically, but I typically undo these changes before committing and pushing.

Example:
| [https://developer.beyondidentity.com](https://developer.beyondidentity.com/docs/next/get-started-nextjs) | [https://developer-docs-git-harry-sc-499-500-beyondidentity.vercel.app](https://developer-docs-git-harry-sc-499-500-beyondidentity.vercel.app/docs/next/get-started-nextjs) |
|--------|--------|
| ![Screenshot 2023-08-28 at 5 19 09 PM](https://github.com/gobeyondidentity/developer-docs/assets/4763133/0f2fa2f4-b63a-44a6-a46b-318806f3a9d4) | ![Screenshot 2023-08-28 at 5 19 04 PM](https://github.com/gobeyondidentity/developer-docs/assets/4763133/e761cae0-3cd4-4e01-88e3-dca252d78e74) |

Reply:
Okay i reverted, but I was wondering - does it look better when it is flush on the left? Pretty much every other instance of it has it left aligned.

![Screenshot 2023-08-29 at 1 36 03 PM](https://github.com/gobeyondidentity/developer-docs/assets/15698998/aff84cd9-cafc-4082-b60b-c9cb5092e6bf)

Reply:
Sorry, I didn't mean to suggest that was the only instance. Patricia was strongly for the indents, aligning everything to the bullets/numbers. I can go either way, but I do think we should be consistent across the docs site, so I've maintained the indents.